Security Group: limit returns in get_bridge_physdev to 1

RPDiep opened the pull request:
## Description
<!--- Describe your changes in detail -->
We've encountered a corner case where `bridge -o link show` returned two lines per bridge instead of one. `get_bridge_physdev` in `security_group.py` returned `bond0.701\nbond0.701`.  
Although this may very well be something on the hypervisor, we should limit the lines returned.
I therefore added a mere `| head -1` to the function.  

See output:  
Old:
```
root@n02:~# bridge -o link show | awk '/master cloudbr0 / && !/^[0-9]+: vnet/ {print $2}'
bond0.701
bond0.701
```
New:
```
root@n02:~# bridge -o link show | awk '/master cloudbr0 / && !/^[0-9]+: vnet/ {print $2}' | head -1
bond0.701
```

<!-- For new features, provide link to FS, dev ML discussion etc. -->
<!-- In case of bug fix, the expected and actual behaviours, steps to reproduce. -->

<!-- When "Fixes: #<id>" is specified, the issue/PR will automatically be closed when this PR gets merged -->
<!-- For addressing multiple issues/PRs, use multiple "Fixes: #<id>" -->
<!-- Fixes: # -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

## Screenshots (if appropriate):

## How Has This Been Tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->
Tested function from idle:
```
root@n02:/usr/share/cloudstack-common/scripts/vm/network# python
Python 2.7.12 (default, Oct  8 2019, 14:14:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from security_group import get_bridge_physdev
>>> get_bridge_physdev('cloudbr0')
'bond0.701'
```

<!-- Please read the [CONTRIBUTING](https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md) document -->

Please let me know if you want me to rebase against 4.13 .

RPDiep commented:
Rebased against 4.13 as requested by @GabrielBrascher

rhtyd commented on scripts/vm/network/security_group.py:
```diff
@@ -149,7 +149,7 @@ def split_ips_by_family(ips):
 
 def get_bridge_physdev(brname):
-    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}'" % brname)
+    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}' | head -1" % brname)
```
Can you rewrite this to use `iproute2`?

RPDiep replied:
Excerpt from commit message:

>     In addition use the 'bridge' command instead of 'brctl'. 'bridge' is part of the iproute2 utils just like 'ip' and should be considered as the new default.

ustcweizhou commented:
@RPDiep @GabrielBrascher 
I did not see this issue on all our platforms. we also use bonding on some platforms.
it seems to be caused by another issue. 
could you please paste result of command "bridge -o link show | awk '/master cloudbr0 /'

RPDiep commented:
@ustcweizhou 

I'm sure this is something now on the hypervisor, but it appears that corner cases exist in which a second line appears. As we only need one result, I added the `| head -1` in this commit.

Here's the output you requested (I added the vnet filter)
```
root@n02:~# bridge -o link show | awk '/master cloudbr0 / && !/^[0-9]+: vnet/'
7: bond0.701 state UP @bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 master cloudbr0 state forwarding priority 32 cost 2 
7: bond0.701 state UP @bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 master cloudbr0 
```

rhtyd commented:
@blueorangutan package

blueorangutan commented:
@rhtyd a Jenkins job has been kicked to build packages. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result: ✔centos6 ✔centos7 ✔debian. JID-325

ustcweizhou commented:
> state forwarding priority 32 cost 2

@RPDiep thanks . this change looks good to me.

rhtyd commented:
Thanks for  confirming the change and tests. I'll merge this.
